other/docker/windows/build_toxcore.sh

```diff
@@ -99,17 +112,25 @@ build()
     do
         ${WINDOWS_TOOLCHAIN}-ar xv ${archive}
     done
+
+    if [ "${CROSS_COMPILE}" = "true" ]; then
+        LIBWINPTHREAD_DIR="/usr/${WINDOWS_TOOLCHAIN}/sys-root/mingw/lib/libwinpthread.a"
```

Comment:
Name ends in `_DIR`, yet contains a path to a file rather than a dir.

Reply:
Done.